templates_jinja2/suggest_match_video_playlist.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Add Match Video{% endblock %}
+
+{% block meta_description %}Add match videos to The Blue Alliance.{% endblock %}
+
+{% block content %}
+<script type="text/javascript">
+    function load_playlist() {
+        var playlist_url = $('#playlist_url').val();
+        var video_list = $('#video_list');
+        var captured = /list=([^&]+)/.exec(playlist_url);
+        var playlist_id = captured ? captured[1] : null;
+        if (playlist_id == null) {
+            video_list.empty();
+            video_list.append($("<p>Invalid playlist URL</p>"));
+            return;
+        }
+        video_list.empty();
+        video_list.append($("<p>Loading videos...</p>"));
+        $.get("/_/yt/playlist/videos", {'playlist_id': playlist_id}).done(function(data) {
+            var video_ids = JSON.parse(data);
+            video_list.empty();
+            video_list.append($("<input>", {name: "num_videos", type: "hidden", value: video_ids.length}));
+            $.each(video_ids, function(index, video_data) {
+                var yt_id = video_data['snippet']['resourceId']['videoId'];
+                var li = $("<li>");
+                var group = $('<div>', {class: 'input-group'});
+                li.append(group);
+                group.append($("<a>", {href:"https://youtu.be/"+yt_id}).append($("<h4>"+video_data['snippet']['title']+"</h4>")));
```

Comment:
You should make this target="_blank" so clicking on a link doesn't wipe all progress.